--- a/src/feature.h	2013-04-07 03:35:12.939530448 +0800
+++ b/src/feature.h	2013-04-07 03:35:07.292893579 +0800
@@ -1272,8 +1272,8 @@
  * +signs		Allow signs to be displayed to the left of text lines.
  *			Adds the ":sign" command.
  */
-#if defined(FEAT_BIG) || defined(FEAT_SUN_WORKSHOP) \
-	    || defined(FEAT_NETBEANS_INTG)
+/* #if defined(FEAT_BIG) || defined(FEAT_SUN_WORKSHOP) \ */
+/* 	    || defined(FEAT_NETBEANS_INTG) */
 #define FEAT_SIGNS
 # if ((defined(FEAT_GUI_MOTIF) || defined(FEAT_GUI_ATHENA)) \
 		&& defined(HAVE_X11_XPM_H)) \
@@ -1281,7 +1281,7 @@
 	|| (defined(WIN32) && defined(FEAT_GUI))
 #  define FEAT_SIGN_ICONS
 # endif
-#endif
+/* #endif */
 
 /*
  * +balloon_eval	Allow balloon expression evaluation. Used with a
